lib/fastlane/plugin/changelog/actions/stamp_changelog.rb

```diff
@@ -57,37 +58,77 @@ def self.stamp(changelog_path, section_identifier, stamp_date, git_tag, placehol
             end
 
             # Output read line
-            file_content.concat(line)
+            file_content.concat(last_line)
```

Comment:
It was not, I changed the logic so that it adds the current line until the Identifier is modified. It will skip that line, since it was already added, and then delay the insertion, to keep the last line for the next part of code...
It's not elegant, but the `.pop()` dit not work on the array...